system-test/spanner.ts

```diff
@@ -6144,6 +6144,186 @@ describe('Spanner', () => {
+      };
+
+      const googleSqlDeleteReturning = {
+        sql:
+          'DELETE FROM ' +
+          TABLE_NAME +
+          ' t WHERE t.KEY = @key ' +
+          'THEN RETURN *',
+        params: {key},
+      };
+
+      const googleSqlDelete = {
+        sql: 'DELETE FROM ' + TABLE_NAME + ' t WHERE t.KEY = @key',
+        params: {key, num},
+      };
+
+      const rowCountRunUpdate = (
+        done,
+        database,
+        insertQuery,
+        updateQuery,
+        deletequery
+      ) => {
+        database.runTransaction((err, transaction) => {
+          assert.ifError(err);
+
+          transaction!
+            .runUpdate(insertQuery)
+            .then(data => {
+              const rowCount = data[0];
+              assert.strictEqual(rowCount, 1);
+              return transaction!.runUpdate(updateQuery);
+            })
+            .then(data => {
+              const rowCount = data[0];
+              assert.strictEqual(rowCount, 1);
+              return transaction!.runUpdate(deletequery);
+            })
+            .then(data => {
+              const rowCount = data[0];
+              assert.strictEqual(rowCount, 1);
+              return transaction!.commit();
+            })
+            .then(() => done(), done)
+            .catch(done);
+        });
+      };
+
+      it('GOOGLE_STANDARD_SQL should return rowCount from runUpdate with dml returning', done => {
+        rowCountRunUpdate(
+          done,
+          DATABASE,
+          googleSqlInsertReturning,
+          googleSqlUpdateReturning,
+          googleSqlDeleteReturning
+        );
+      });
+
+      const assertRowsAndRowCount = data => {
+        const rows = data[0];
+        const stats = data[1];
+        const rowCount = Math.floor(stats[stats.rowCount!] as number);
+        assert.strictEqual(rowCount, 1);
+        rows.forEach(row => {
+          const json = row.toJSON();
+          assert.strictEqual(json.Key, key1);
+          assert.strictEqual(json.StringValue, str);
+        });
+      };
+
+      const rowCountRun = (
+        done,
+        database,
+        insertQuery,
+        updateQuery,
+        deletequery
+      ) => {
+        database.runTransaction((err, transaction) => {
+          assert.ifError(err);
+
+          transaction!
+            .run(insertQuery)
+            .then(data => {
+              assertRowsAndRowCount(data);
+              return transaction!.run(updateQuery);
+            })
+            .then(data => {
+              assertRowsAndRowCount(data);
+              return transaction!.run(deletequery);
+            })
+            .then(data => {
+              assertRowsAndRowCount(data);
+              return transaction!.commit();
+            })
+            .then(() => done(), done)
+            .catch(done);
+        });
+      };
+
+      it('GOOGLE_STANDARD_SQL should return rowCount and rows from run with dml returning', done => {
+        googleSqlInsertReturning.params.key = key1;
```

Comment:
This can be written in a single line as `googleSqlInsertReturning.params.key=googleSqlUpdateReturning.params.key=googleSqlDeleteReturning.params.key=key1`

Reply:
Removed different keys for different tests as it was working with same key.